File: app/src/main/java/com/draglabs/dsoundboy/dsoundboy/Interfaces/CallbackListener.java
package com.draglabs.dsoundboy.dsoundboy.Interfaces;

/**
 * Created by davrukin on 9/18/17.
 * Tutorial link: https://stackoverflow.com/a/18279548
 */

public interface CallbackListener {

    // AuthenticateUser
    void uniqueUserIDset();

    // StartJam and ExitJam
    void jamPINset();
    void jamIDset();
    void jamStartTimeSet();
    void jamEndTimeSet();

    // GetCollaborators
    void getCollaboratorsSet();

    // GetUserActivity
    void getUserActivitySet();

    // GetJamDetails
    void getJamDetailsSet();

    // GetAccessToken
    void getAccessTokenSet();
}

File: app/src/main/java/com/draglabs/dsoundboy/dsoundboy/Interfaces/CallbackListener.kt
package com.draglabs.dsoundboy.dsoundboy.Interfaces

/**
 * Created by davrukin on 9/18/17.
 * Tutorial link: https://stackoverflow.com/a/18279548
 * @author Daniel Avrukin
 */

@Deprecated("No longer needed")
interface CallbackListener {

    // AuthenticateUser
    fun uniqueUserIDset()

    // StartJam and ExitJam
    fun jamPINset()

    fun jamIDset()
    fun jamStartTimeSet()
    fun jamEndTimeSet()

    // GetCollaborators
    fun getCollaboratorsSet()

    // GetUserActivity
    fun getUserActivitySet()

    // GetJamDetails
    fun getJamDetailsSet()

    // GetAccessToken
    fun getAccessTokenSet()
}
